src/s2mMuscleHillTypeThelenFatigable.cpp

```diff
@@ -63,6 +63,19 @@ void s2mMuscleHillTypeThelenFatigable::computeFlCE(const s2mMuscleStateActual &E
     m_FlCE *= m_fatigueState->activeFibers();
 }
 
+std::shared_ptr<s2mMuscleFatigueState> s2mMuscleHillTypeThelenFatigable::fatigueState()
+{
+    return m_fatigueState;
+}
+
+s2mVector s2mMuscleHillTypeThelenFatigable::getDerivativeState()
```

Comment:
Is there a particular reason to create an interface for derivative state in here? I feel that it should be the user sake to do that from the fatigueState itself. The danger by doing this, is if we create a new state some day, is to forget this interface even exists... Therefore, it won't evolves with the code. 
But I can see how it can be interesting... What was the purpose?

In any case, it would be in the wrong file... It definitely belongs to s2mMuscleFatigue and not s2mMuscleHillTypeThelenFatigable